toss_3_x86_64_ib/spack.yaml

```diff
@@ -0,0 +1,728 @@
+spack:
+
+  config:
+    install_tree:
+      root: $spack/..
+      projections:
+        all: "{compiler.name}-{compiler.version}/{name}-{version}-{hash}"
+    # Note: unlike projects using RADIUSS Spack
+    # Configs, Serac is using "$spack/.." as the root
+    # of Spack cache directories. We share the goal of
+    # making sure that Spack instances do not collide,
+    # our choice supposes that $user_cache_path is
+    # defined to that purpose, but Serac is enforcing
+    # it for the cache directories.
+    # The reason for not adopting the same strategy as
+    # Serac is because there are other directories
+    # created by Spack that need to be porperly
+    # isolated. Therefore, setting the
+    # "user_cache_path" properly isstill mandatory.
+    build_stage::
+      - $user_cache_path/stage
+    test_stage: $user_cache_path/test
+    misc_cache: $user_cache_path/misc
+    flags:
+      keep_werror: 'all'
+
+    view: false
+
+  compilers::
+  - compiler:
```

Comment:
Similar question to the one I asked about old versions in p[roject Spack packages: should we remove old compiler versions that are no longer available?  For example, on TOSS4 systems, clang/14.0.6 appears to be the only version available now.

Reply:
We can definitely do some clean up here. Let me open an issue for that.